### ISSUE TITLE
VerifyingRelocationHelper - No data available

### ISSUE THREAD

**Issue by AlessioDP** [slimjar/slimjar#35]
Hello,
an user reported me this error in his MC server console:
```
[13:07:40 WARN]: com.alessiodp.parties.core.common.addons.external.slimjar.injector.InjectionFailedException: SlimJar failed to inject dependency: name -> Dependency{groupId='com.h2database', artifactId='h2', version='1.4.200', snapshotId='null', transitive=[]}
[13:07:40 WARN]: at com.alessiodp.parties.core.common.addons.external.slimjar.injector.SimpleDependencyInjector.injectDependencies(SimpleDependencyInjector.java:63)
[13:07:40 WARN]: at com.alessiodp.parties.core.common.addons.external.slimjar.injector.SimpleDependencyInjector.inject(SimpleDependencyInjector.java:50)
...
...
[13:07:40 WARN]: Caused by: java.nio.file.FileSystemException: /hlserver/27897/plugins/Parties/libraries/com/h2database/h2/1.4.200/relocated/Parties/h2-1.4.200.jar: Unable to get size of extended attribute 'slimjar.owner': No data available
[13:07:40 WARN]: at java.base/sun.nio.fs.LinuxUserDefinedFileAttributeView.size(LinuxUserDefinedFileAttributeView.java:153)
[13:07:40 WARN]: at com.alessiodp.parties.core.common.addons.external.slimjar.relocation.meta.AttributeMetaMediator.readAttribute(AttributeMetaMediator.java:43)
[13:07:40 WARN]: at com.alessiodp.parties.core.common.addons.external.slimjar.relocation.helper.VerifyingRelocationHelper.relocate(VerifyingRelocationHelper.java:68)
[13:07:40 WARN]: at com.alessiodp.parties.core.common.addons.external.slimjar.injector.helper.InjectionHelper.fetch(InjectionHelper.java:49)
[13:07:40 WARN]: at com.alessiodp.parties.core.common.addons.external.slimjar.injector.SimpleDependencyInjector.injectDependencies(SimpleDependencyInjector.java:56)
[13:07:40 WARN]: ... 19 more
```

It seems it started to do that after some time that the libraries folder has been created, deleting that folder and restarting the server resolved the issue.
What can be the root cause?

**Comment by Vshnv** [comment]
Looks like something removed the xattr's for the original file's hash from the file or it was overwritten. Ill try to replicate the bug on my end and get a fix up asap.

**Comment by Vshnv** [comment]
Fixed in 1.2.4+
